base/libc.jl

```diff
@@ -437,6 +437,33 @@ function srand(seed::Integer=_make_uint64_seed())
     ccall(:jl_srand, Cvoid, (UInt64,), seed % UInt64)
 end
 
+"""
+    mkfifo(path::AbstractString, [mode::Integer]) -> path
+
+Make a FIFO special file (a named pipe) at `path`.  Return `path` as-is on success.
+
+`mkfifo` is supported only in Unix platforms.
+
+!!! compat "Julia 1.8"
```

Comment:
Has this been corrected?  Obviously this merge was *a bit* late for 1.8.

Reply:
Good catch. Please PR to fix